### DISCUSSION TITLE
Welcome to chess Discussions!

### DISCUSSION THREAD

**Discussion by notnil** [notnil/chess#61] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by i-jonathan** [comment]
Hello @notnil 
I was trying to use this library and didn't find an easy way to check if a move puts the opponent in check, or even to check if my king is on Check. 
Is there a way to go about this?

**Reply by notnil** [reply]
[move](https://pkg.go.dev/github.com/notnil/chess#Move)

@i-jonathan move.HasTag(chess.Check) should do the trick!

**Reply by i-jonathan** [reply]
Nice. Thank you. 
Though that adds a bit of complexity when working with Movestr. Cause the move isn't returned by Movestr method

**Reply by i-jonathan** [reply]
Hiyo @notnil 
I'm back again. I'm wondering if there's a way to skip or end a turn. Or anything of that sort really
